jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-api/src/main/java/org/jbpm/console/ng/wi/dd/model/DeploymentDescriptorModel.java

```diff
@@ -245,35 +245,20 @@ public boolean equals(Object o) {
     @Override
     public int hashCode() {
         int result = persistenceUnitName != null ? persistenceUnitName.hashCode() : 0;
-        result = ~~result;
```

Comment:
Is it ok to delete these `result = ~~result;` ? According to this [stackoverflow answer](http://stackoverflow.com/questions/29736952/what-is-the-meaning-of-double-tilde-in-java) they might make a difference..